docs/translating.md

```diff
@@ -58,6 +58,14 @@ You can instead use:
 {% blocktrans %}This string will have {{ value }} inside.{% endblocktrans %}
 ```
 
+## Frontend Translations
+The first step in translating the frontend is to create the translation
+files using `./tools/make-frontend-messages`. This command will create
+translation files under `static/locale`, the location can be changed by
+passing an argument to the command, however make sure that the location is
+publically accessible since these files are loaded through XHR in the
+frontend which will only work with publically accessible resources.
+
```

Comment:
OK; I think it might be reasonable to just have two translation files that we maintain in Transifex, one for each piece, as long as we can make the tooling such that folks don't have to think about it.